### ISSUE TITLE
Range mode ?

### ISSUE THREAD

**Issue by bosquig** [kenhyuwa/vue-tailwind-picker#6]
Hey,

Would be great if you could add a date range picker mode and a clear date button (also, a nullable start-date)

**Comment by kenhyuwa** [comment]
👋 thanks for the suggestion, maybe for the next version update.

**Comment by ljcremer** [comment]
I agree this will be great!

**Comment by FPMedia** [comment]
Yes please! Is this scheduled yet? Thanks so much for this package - it's really great.

**Comment by ryarturogi** [comment]
Went this feature is coming?
Would be great to have this implemented.
What do you need to add this sooner?

**Comment by kenhyuwa** [comment]
Please, move to [Litepie Datepicker](https://github.com/kenhyuwa/litepie-datepicker) for Vue3.
